roles/kubernetes-apps/registry/templates/registry-svc.yml.j2

```diff
@@ -8,10 +8,27 @@ metadata:
     k8s-app: registry
     addonmanager.kubernetes.io/mode: Reconcile
     kubernetes.io/name: "KubeRegistry"
+{% if registry_service_annotations %}
+  annotations:
+{% for key, value in registry_service_annotations.items() %}
```

Comment:
I would rewrite this using `to_nice_yaml(indent=2)`.

Reply:
thanks :+1: